### PR TITLE
Adding clapit to showcase

### PR DESCRIPTION
**Pull request by ewindso**
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

(You can skip this if you're fixing a typo or adding an app to the Showcase.)

Explain the **motivation** for making this change. What existing problem does the pull request solve?

Prefer **small pull requests**. These are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Make sure tests pass on both Travis and Circle CI.

**Code formatting**

Look around. Match the style of the rest of the codebase. See also the simple [style guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#style-guide).

For more info, see the ["Pull Requests" section of our "Contributing" guidelines](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests).

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -212,6 +212,12 @@ var featured = [
...
   },
+  {
+    name: 'clapit',
+    icon: 'http://a4.mzstatic.com/us/r30/Purple60/v4/1a/42/5b/1a425b56-848a-91f5-8078-9f5473c9021f/icon350x350.png',
+    link: 'https://itunes.apple.com/us/app/clapit/id1062124740?mt=8',
+    author: 'Refined Edge Solutions, LLC'
+  }
 ];
 
 var apps = [
```
Comment (ide) [inline website/src/react-native/showcase.js]: Move the JSON entry to the apps section and keep alphabetical order

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by facebook-github-bot-6** [comment]
@ewindso updated the pull request.

File: website/src/react-native/showcase.js (hunk 1 above)
Comment (ewindso) [inline website/src/react-native/showcase.js]: @ide yes sir, done!

**Comment by facebook-github-bot-9** [comment]
@ewindso updated the pull request.

**Comment by brentvatne** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/644664045680824/int_phab) to review.

**Pull request closed without merging**